pkg/libpciaccess

```diff
@@ -1,16 +1,14 @@
 [mirrors]
-http://xorg.freedesktop.org/archive/individual/lib/libpciaccess-0.14.tar.bz2
+http://www.x.org/releases/individual/lib/libpciaccess-0.17.tar.xz
 
 [vars]
-filesize=361375
-sha512=bf40214dfd22f2a223f7c948566e6eaa4bfed60116e71b3eb19f6fe7ab2c4546649eb0416c06d5983e476ea5e27de036936646eb5f800ea97763b6d1f1976f4b
-pkgver=3
+filesize=331768
+sha512=8484605c66ef18c8d8a3e029a6d33e26fcaa450c1263790d89ac8b0a160ffb2dfceaa6140ac9ad7d8087283ecbec8ac801f757e44890d547c3cbecd2c922ca47
+pkgver=4
 
 [deps]
 
 [build]
-patch -p1 < "$K"/libpciaccess_PATH_MAX.patch
-patch -p1 < "$K"/libpciaccess-arm.patch
```

Comment:
if those are no longer required, the commit bumping them should also remove them